cmgd/cmgd_db.c

```diff
@@ -387,14 +387,10 @@ static int cmgd_walk_db_nodes(cmgd_db_ctxt_t *db_ctxt,
 		assert(xpath);
 		CMGD_DB_DBG(" -- XPATH: %s", xpath);
 
-		/* 
-		 * NOTE: This specific child will be visited from the next call to 
-		 * cmgd_walk_db_nodes().
-		 */
-		// if (iter_fn) {
-		// 	(*iter_fn)((cmgd_db_hndl_t) db_ctxt, xpath,
-		// 		dnode, nbnode, ctxt);
-		// }
+		if (iter_fn) {
```

Comment:
This code is going to affect other code flows as well. Did you test all of that?